src/pages/Boards/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Boards = () => {
+  return (
+    <div>
+      Boards
+    </div>
+  );
+};
+
+export default Boards;
```

Comment:
A newline on the end of file is required, make sure your files are already add a newline on the end.